nginx/nginx.conf

```
@@ -44,10 +44,12 @@ http {
         ssl_certificate             /etc/nginx/private/cert.pem;
         ssl_certificate_key         /etc/nginx/private/key.pem;
         ssl_prefer_server_ciphers   on;
```

Mozilla ("intermediate", nginx `1.19.6`, openssl `1.1.1g`) [suggest](https://ssl-config.mozilla.org/#server=nginx&version=1.19.6&config=intermediate&openssl=1.1.1g&guideline=5.7) `off` as does rando [SF post](https://serverfault.com/questions/997614/setting-ssl-prefer-server-ciphers-directive-in-nginx-config):

```suggestion
        ssl_prefer_server_ciphers   off;
```

---

I can't say I agree here. This has always been `on` in this config. Only reason to turn it `off` seems to be to support older clients. Given we've added the ciphers back, I think it makes sense to leave it `on`.